sdk/communication/communication-sms/test/smsClientWithToken.spec.ts

```diff
@@ -32,7 +36,7 @@ describe("SmsClientWithToken [Playback/Live]", async () => {
       this.skip();
     }
 
-    const endpoint = env.COMMUNICATION_ENDPOINT;
+    const endpoint = communicationEndpoint();
```

Comment:
Please use the existing util instead of string splitting the connection string yourself
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/communication/communication-common/src/credential/connectionString.ts

```suggestion
    const endpoint = parseConnectionString(env.AZURE_COMMUNICATION_LIVETEST_CONNECTION_STRING).endpoint;
```

Reply:
Nice. Completely forgot this method existed in the common package.

Reply:
Made the changes. However the import statement looks a little verbose. 
`import { parseConnectionString } from "../../communication-common/src/credential/connectionString";`
Is there a nicer way to import this?